BioArchLinux/rstudio-desktop/PKGBUILD

```diff
@@ -0,0 +1,138 @@
+# Maintainer: Artem Klevtsov <a.a.klevtso@gmail.com>
+# Maintainer: Stephen Martin <hwkiller@gmail.com>
+# Contributor: Conor Anderson <conor@conr.ca>
+
+pkgname=rstudio-desktop
+pkgver=2023.06.2+561
+_pkgver=${pkgver//+/.}
+_srcname=rstudio-${pkgver//+/-}
+_gitcommit=${_get_commit_from_tag}
+_gitname=rstudio-rstudio-${_gitcommit}
+_gwtver=2.8.2
+_ginver=2.1.2
+_nodever=16.14.0
+_pandocver="current"
+_quarto="FALSE"
+pkgrel=1
+pkgdesc="A powerful and productive integrated development environment (IDE) for R programming language"
+arch=('x86_64')
+url="https://posit.co/products/open-source/rstudio/"
+license=('AGPL3')
+depends=('r>=3.3.0' 'boost-libs' 'qt5-sensors' 'qt5-svg' 'glibc' 'zlib' 'bash' 'gcc-libs' 'qt5-webengine' 'qt5-xmlpatterns' 'postgresql-libs' 'sqlite3' 'soci' 'clang' 'hunspell-en_US' 'mathjax2' 'pandoc' 'yaml-cpp' 'r-rmarkdown' 'r-renv' 'r-testthat' 'r-xml2' 'r-yaml')
+makedepends=('git' 'cmake>=3.1.0' 'boost' 'desktop-file-utils' 'jdk8-openjdk' 'apache-ant' 'unzip' 'openssl' 'libcups' 'pam' 'patchelf' 'wget' 'yarn')
+optdepends=('git: for git support'
+            'subversion: for subversion support'
+            'openssh-askpass: for a git ssh access'
+            'quarto: for Quarto projects support')
+
+provides=('rstudio-desktop')
+conflicts=('rstudio-desktop' 'rstudio-desktop-bin' 'rstudio-desktop-preview' 'rstudio-desktop-git')
+source=("rstudio-$_pkgver.tar.gz::https://github.com/rstudio/rstudio/archive/refs/tags/v2023.06.2+561.tar.gz"
+        "https://github.com/quarto-dev/quarto/archive/refs/heads/release/rstudio-cherry-blossom.zip"
+        "https://nodejs.org/dist/v${_nodever}/node-v${_nodever}-linux-x64.tar.gz"
+        "qt.conf"
+        "pandoc_version.patch")
+
+sha256sums=('83469a4b7311d745c24d7d9e55c44aa4325dda3bdde50a903e2f9ead47d75cc7'
+            'df3a040f0cf4ce1892519082dd6822a8ca433e9e4f02d9394ab2931733f7e5a6'
+            '2c69e7b040c208b61ebf9735c63d2e5bcabfed32ef05a9b8dd5823489ea50d6b'
+            '723626bfe05dafa545e135e8e61a482df111f488583fef155301acc5ecbbf921'
+            '286925c442c1818979714feeec1577f03ae8a3527d2478b0f55238e2272a0b9e')
+
+noextract=("gin-${_ginver}.zip")
```

Comment:
There is no source file with this name, so this line is unnecessary.

Reply:
removed